test/integration/test_dwarf.c

```diff
@@ -1430,6 +1430,68 @@ bool test_dwarf5_loclists(void) {
 	mu_end;
 }
 
+bool test_dwarf4_loclists(void) {
+	RzBin *bin = rz_bin_new();
+	RzIO *io = rz_io_new();
+	rz_io_bind(io, &bin->iob);
+
+	RzBinOptions opt = { 0 };
+	rz_bin_options_init(&opt, 0, 0, 0, false);
+	RzBinFile *bf = rz_bin_open(bin, "bins/pe/vista-glass.exe", &opt);
```

Comment:
Judging by this test, now it works even for PE files, right? https://github.com/rizinorg/rizin/issues/3581

Reply:
Yes